<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskDialog</class>
 <widget class="QDialog" name="TaskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建任务</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tWTask">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>任务信息</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QGroupBox" name="gBTaskinfo">
           <property name="maximumSize">
            <size>
             <width>248</width>
             <height>265</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="lebelStore">
                  <property name="text">
                   <string>零售店：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="combStore"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>服务类型:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="combServType"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>用户名：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lEUserName"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>密码：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lEPasswd">
                  <property name="echoMode">
                   <enum>QLineEdit::Password</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>身份证号：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lEGovId"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>手机号：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lEPhoneNumber"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>预约码：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lEReservCode"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QPushButton" name="pBStartTask">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="text">
                   <string>开始</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pBClear">
                  <property name="text">
                   <string>清空</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="gBTaskProcessinfo">
           <property name="maximumSize">
            <size>
             <width>207</width>
             <height>265</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QSplitter" name="splitter">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <widget class="QWidget" name="">
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QProgressBar" name="progressBar">
                  <property name="value">
                   <number>24</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="groupBox">
                  <property name="title">
                   <string>单击刷新</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <item>
                    <widget class="QLabel" name="lbVerifyCodePic">
                     <property name="text">
                      <string>TextLabel</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>验证码：</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lEVerfiyCode"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QPushButton" name="pBOk">
                    <property name="text">
                     <string>确定</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pBCancel">
                    <property name="text">
                     <string>取消</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
           <zorder>splitter</zorder>
           <zorder></zorder>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lEVerfiyCode</tabstop>
  <tabstop>combStore</tabstop>
  <tabstop>lEPasswd</tabstop>
  <tabstop>lEPhoneNumber</tabstop>
  <tabstop>lEReservCode</tabstop>
  <tabstop>pBCancel</tabstop>
  <tabstop>pBOk</tabstop>
  <tabstop>lEUserName</tabstop>
  <tabstop>lEGovId</tabstop>
  <tabstop>pBClear</tabstop>
  <tabstop>pBStartTask</tabstop>
  <tabstop>combServType</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pBStartTask</sender>
   <signal>clicked()</signal>
   <receiver>TaskDialog</receiver>
   <slot>startTask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startTask()</slot>
 </slots>
</ui>
